main/src/cgeo/geocaching/utils/CalendarUtils.java

```diff
@@ -162,4 +164,13 @@ public static long parseDayMonthYearUS(final String date) {
 
         return new ImmutablePair<>(date.getTimeInMillis(), date.getTimeInMillis() + 86400000);
     }
+
+    /**
+     * Generate a time zone string for the users system time config
+     */
+    public static String getUserTimeZoneString() {
+        final TimeZone timeZone = TimeZone.getDefault();
+        final int timeZoneGMTOffset = timeZone.getRawOffset();
+        return timeZone.getDisplayName(false, TimeZone.SHORT) + " / " + (timeZoneGMTOffset < 0 ? "-" : "+") + TimeUnit.HOURS.convert(timeZoneGMTOffset, TimeUnit.MILLISECONDS);
```

Comment:
Please change this to output ".../UTC+x".
This is easier to understand later.

Your output says MEZ. Should't it be MESZ, or better non localized MEST?

Reply:
Here we go. Complex requests result in a simple solution... Codewise much cleaner and it will print "GMT+02:00" in this example which is the US ISO standard.

Reply:
Hmm, now it is too simple .
Why not stay with both parts and use for the first one `getDisplayName(boolean daylight, int style, Locale locale)` with `daylight = true` and `locale = Locale.US` ?

Reply:
> with `daylight = true`

This will then result in that daylight is always applied, independent of whether it is used in that time zone at all. 

So an additional check would be necessary, which however IMHO get much to complex. 

But why should we ever need both formats in the system information?